### ISSUE TITLE
GAN model cannot converge.

### ISSUE THREAD

**Issue by Bleachzer** [vikigenius/conditional_text_generation#20]
I found that the GAN model cannot converge during training. I followed the data format in tests/fixtures folder to create the train_setences.tsv file and train_dialog.tsv file from the de-duplicated version of DailyDialog dataset which is proposed by Bahuleyan et al. in their GitHub [https://github.com/HareeshBahuleyan/probabilistic_nlg/tree/master/dialog/data/DailyDial/de_duplicated](url). I did not change any code and the VAE model is well trained whereas the GAN model will early stop after training 5 epochs. The training loss for GAN model did not decrease and the GAN model can only produce some meaningless sentences.

Are there any suggestions for solving this?
